docs/exult_intrinsics.txt

```diff
@@ -3237,6 +3237,12 @@ obj->obj_sprite_effect(int sprite, int rel_x, int rel_y, int vel_x, int vel_y,
               the sprite frame will go through all of its frames. If negative,
               the sprite will go through all of its frames *once*.
 
+string UI_play_scene(string text)
```

Comment:
Please note that this file is generated. In the exult-web repository, I add a "header" for the intrinsic on `seventowers/usecode/intrinsics/defs`, add the documentation body for the intrinsic on `seventowers/usecode/intrinsics`, add the intrinsic under an appropriate group on `seventowers/usecode/reference/intrinsic_relations.dat` (including adding links to related intrinsics), and add it to `seventowers/geshi/ucc.php` for syntax highlighting.

I then generate the doc for Exult through

```bash
$ cd /path/to/exult-web/seventowers
$ php code.php TITLE_IMAGE=usecodetitle.png DATAFILE=exult_intrinsics.dat TYPE=1 OUTPUT=text > /path/to/exult/docs/exult_intrinsics.txt
```